app/src/controller.c

```diff
@@ -48,6 +48,10 @@ controller_destroy(struct controller *controller) {
 bool
 controller_push_msg(struct controller *controller,
                       const struct control_msg *msg) {
+    if (sc_get_log_level() == SC_LOG_LEVEL_VERBOSE) {
```

Comment:
I'd use `(sc_get_log_level() <= SC_LOG_LEVEL_VERBOSE)` even if we don't expect any lower log levels.

Reply:
:+1: